src/vs/workbench/browser/parts/panel/panelPart.ts

```diff
@@ -27,13 +27,15 @@ import { CompositeBar } from 'vs/workbench/browser/parts/compositeBar';
 import { ToggleCompositePinnedAction } from 'vs/workbench/browser/parts/compositeBarActions';
 import { IBadge } from 'vs/workbench/services/activity/common/activity';
 import { INotificationService } from 'vs/platform/notification/common/notification';
-import { Dimension } from 'vs/base/browser/dom';
+import { Dimension, trackFocus } from 'vs/base/browser/dom';
 import { localize } from 'vs/nls';
 import { IDisposable } from 'vs/base/common/lifecycle';
 import { RawContextKey, IContextKey, IContextKeyService } from 'vs/platform/contextkey/common/contextkey';
 
 const ActivePanleContextId = 'activePanel';
```

Comment:
I do not see the point of these consts, `ActivePanleContextId` and `PanelFocusContextId` why don't we simply inline them, we are using them only once?